subsys/bluetooth/controller/Kconfig

```diff
@@ -6,6 +6,20 @@
 
 menu "Nordic BLE controller"
 
+config BLECTRL_MASTER_COUNT
```

Comment:
I have added two options to control master and slave roles count. Also set the numbers accordingly for each device. I am able to get two devices to connect to dongle but connection is not stable. It breaks after a minute or two. Usually for single device.

Reply:
Thanks for adding the configuration option. Could you provide some more details about the issue you are observing? "The connection breaks" is not giving enough information to be able to debug the problem

Reply:
> Thanks for adding the configuration option. Could you provide some more details about the issue you are observing? "The connection breaks" is not giving enough information to be able to debug the problem

Sure, I know this is not very specific... I have not yet spent any time to look what is wrong. The effect is that when I start dongle it finds one device quickly. Then when it scans for the second one it finds it and connects. The secure does not happen however for couple of seconds (significantly longer then for the first device). Then both device work fine. I am able to type and move cursor around. I get 500 reports per seconds. After a minute or two keyboard disconnects - I have not seen anything specific in the logs.
```
[06101716] <inf> ble_state: Disconnected from f3:10:8b:db:8b:a4 (random) (reason 8)
[06101732] <inf> event_manager: e: ble_peer_event id=0x20003c64 DISCONNECTED
```
Dongle looks for keyboard again but this time crashes when parsing gatt characteristics.

Reply:
The crash occurring when parsing gatt characteristics, described in https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1279#issuecomment-542216745 is likely a bug in gatt_dm

Reply:
Yes. We will leave it for now and track it later. This change does not kill CI and we will need to deal with it here.